scripts/import-expansions.js

```diff
@@ -27,10 +27,17 @@ const importExpansions = () => {
   const data = fs.readFileSync(expansionsPath, 'utf8');
   const expansions = YAML.parse(data);
   Object.keys(expansions).forEach((key) => {
+    // we need to rename signing auth token to NOTARY_TOKEN
+    // this is used for notary signing when uploading to barque
+    if (key === 'signing_auth_token_44') {
```

Comment:
I'd assume so since above @lrlna said:
> we will have to adjust the NOTARY_TOKEN and NOTARY_KEY_NAME, as those only work for 4.4.0 series

I think as a first round, I'd be ok with just publishing to`4.4`-`org`. However, soon enough I think we should publish to all 4+ repos and once we go GA also to `enterprise`.

Reply:
Yep, definitely have to update this every time there is a new server version. We will also have to update keys on evergreen and those need to obtained from BUILD team as they manage those tokens. Additionally, if we are supporting mutliple versions, we will have to support and store mutliple tokens.

From what I can tell, all the teams have to go through this process.

Reply:
👍. I would love to have a bit more context here, at a first glance it looks a lot like an unnecessary pain / something that could be handled differently (not talking about your implementation). Maybe I'll bug you later and try to see if there is something we could ask to coordinate a bit better, for example it would be enough to have a private repo or s3 bucket with all the token / key names that someone would update once and for everyone.

For the scope of this PR, if i've got correctly what is the use of that environment variable .. and considering what max said, I think would be better to do this translation where we use it:

``` js
await exec('${curatorDirPath}/curator ...', {
  env: {
     NOTARY_KEY_NAME: 'server-4.4',
     NOTARY_TOKEN: process.env.SIGNING_AUTH_TOKEN_44
  }
});
```

That would also prevent issues if by accident we would define a `notary_token` expansion, which would override or be overridden the other one.

Reply:
@mcasimir I added your suggestion to this PR. I think it works well, but lmk if you were intending for something else.

Reply:
The idea is for `import-expansions` to really know nothing of this curator topic. 

I'd completely remove this `if` and use directly `NOTARY_TOKEN: process.env.SIGNING_AUTH_TOKEN_44` in `exec`. Also because that is where we now have `server-4.4` which i'd assume goes in pair with this one.

Reply:
ah i see! I did misunderstand a little bit then. I'll fix it up to have `NOTARY_TOKEN: process.env.SIGNING_AUTH_TOKEN_44` in `barque.ts`.